src/services/userService.js

```diff
@@ -14,25 +14,55 @@ const getUserSignUp = async (
   gender,
   password,
   profileImg,
-  age
+  age,
+  admin
 ) => {
   validateId(userId);
   validateNickname(nickname);
   validatePassword(password);
 
   const saltRounds = 10;
   const hashedPw = await bcrypt.hash(password, saltRounds);
-
+  
   return await userDao.createUser(
     userId,
     nickname,
     gender,
     hashedPw,
     profileImg,
-    age
+    age,
+    admin
   );
 };
 
+const signIn = async ( userId, password ) => {
+  const user = await userDao.signIn(userId);
+  const checkHash = async (password, hashedPassword) => {
+      return await bcrypt.compare(password, hashedPassword);
+  }
+
+  const checkPassword = await checkHash(password, user.password);
```

Comment:
단순히 bcrypt.compare 함수를 checkHash 함수로 감쌓는데, 오히려 읽어야되는 코드 길이만 길어진 것 같습니다.
함수를 모듈화해서 다른 곳에서도 사용할 목적이 아니라면, `bcrypt.compare(password, hashedPassword);`를 그대로 사용하는 것이 좋을 것 같습니다.

```suggestion
  isMatched = await bcrypt.compare(password, user.password);
  
  if(!isMatched) {...}
```

Reply:
제시해주신 내용대로 수정했습니다